### ISSUE TITLE
Undefined function or variable'xavier'！

### ISSUE THREAD

**Issue by LeiJiangJNU** [anilbas/3DMMasSTN#6]
hi @anilbas,
Thank you for your contribution!
I followed your readme to run the code,but I can't run the training code.Could you please give me some advice on the questions below?
In dagnn_3dmmasstn_init.m line 7. This xavier(1,1,4096,16) should be the algorithm that initializes the network weight, but I run the prompt without definition.
Thanks in advance!

**Comment by suvojit-0x55aa** [comment]
Any update on this ?

**Comment by suvojit-0x55aa** [comment]
Found the [xavier.m](https://github.com/vedaldi/practical-cnn-reg/blob/master/xavier.m) file in this repo.

**Comment by ReverseSystem001** [comment]
> Found the [xavier.m](https://github.com/vedaldi/practical-cnn-reg/blob/master/xavier.m) file in this repo.

hi do you know where is the BFM_UV.mat come from? does if from the result of Output grid which mentioned in the paper 2.3

**Comment by suvojit-0x55aa** [comment]
@ReverseSystem001 yes it is as confirmed here https://github.com/anilbas/3DMMasSTN/issues/8#issuecomment-414996312

**Comment by ReverseSystem001** [comment]
@suvojit-0x55aa thanks for your reply ! I will try it as the paper mentioned
